src/nvim/path.c

```diff
@@ -1098,7 +1102,7 @@ int gen_expand_wildcards(int num_pat, char_u **pat, int *num_file,
     p = pat[i];
 
     if (vim_backtick(p))
-      add_pat = expand_backtick(&ga, p, flags);
+      add_pat = (int)expand_backtick(&ga, p, flags);
```

Comment:
> why wouldn't now?

Because `SIZE_MAX` is more than `INT_MAX`, at least on my system, and the variable inside `expand_backtick` (`cnt`) in addition to the return value has been changed to a `size_t`. From a practical point of view, It's probably almost never an issue.